.env.example

```
@@ -82,6 +82,14 @@ WEAVIATE_API_KEY=
 QDRANT_API_KEY=
 
 
+############
+# [required]
+# Ollama API Caddy Bearer token (auto-generated)
+# Requests to OLLAMA_HOSTNAME must send: Authorization: Bearer <this value>
+############
+OLLAMA_CADDY_API_TOKEN=
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)